components/button/test/button-subtle.visual-diff.html

```diff
@@ -14,22 +14,53 @@
 </head>
 <body class="d2l-typography">
 	<div class="visual-diff">
-		<d2l-button-subtle id="normal" text="Subtle Button"></d2l-button-subtle>
```

Comment:
I'd suggest renaming the golden filenames to match what they'll be after this change. That way the visual diff changes will be more realistic.